### PR TITLE
[12.0][REM] coopaname_custom: hide timezone and twitter on events

### PR DESCRIPTION
**Pull request by mclaeysb**
Small cosmetic change. 

FYI: The twitter hashtag is defined in the original event view as 

```xml
<label for="twitter_hashtag"/>
<div>
    <span class="oe_inline"># </span>
    <field name="twitter_hashtag" nolabel="1" class="oe_inline"/>
</div>
```

### CODE REVIEW

**Review by robinkeunen — APPROVED** [review]
👍 je développe sur la même branche pour simplifier les release.

**Comment by robinkeunen** [comment]
@mclaeysb ready for your reviews (commit by commit to make it easy)

**Comment by robinkeunen** [comment]
@mclaeysb une fois mergé, je propose de bouger le module coopaname_custom vers le repo cie-custom

File: hr_cae_event/models/event.py [hunk 1]
```diff
@@ -28,6 +28,7 @@ class Event(models.Model):
         store=True,
         help="hours",
     )
+    co_organizer_id = fields.Many2one("res.partner", string="Co-Organizer")
```
Comment (mclaeysb) [inline hr_cae_event/models/event.py:31]: The spec wasn't really as clear on this everywhere but the intention what that this be a M2O field. See "Nous pourrions ainsi avoir plus de deux co-organisateurs".

File: hr_cae_event/views/event.xml [hunk 2]
```diff
@@ -11,6 +11,7 @@
         <field name="arch" type="xml">
 
             <field name="organizer_id" position="after">
+                <field name="co_organizer_id"/>
```
Comment (mclaeysb) [inline hr_cae_event/views/event.xml:14]: O2M > with tags

File: coopaname_custom/views/event_registration.xml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    </record>
+
+    <menuitem name="Registrations"
+              id="event.menu_action_registration"
+              parent="event.menu_reporting_events"
+              sequence="4"
+              action="event.action_registration"
+              groups="event.group_event_manager"/>
+
+</odoo>
```
Comment (mclaeysb) [inline coopaname_custom/views/event_registration.xml:47]: A bit of guessing work (and learning) what's happening here. But from a code perspective this LGTM!

**Review by mclaeysb — COMMENTED** [review]
Great code!

Reading the spec of AP1b again, I see that it contains the historical reasoning, with the conclusion in the last paragraph: adding (co-)organisers to the list of attendees in order to manage their timesheets is not done by default, but can be done manually if needed. This would mean that your 1a60889 commit, although great code, would be dropped. Let's check again though, because maybe we'll keep it now that it's coded. 

And let's make sure our specs are clear enough - we missed a beat there this time...

File: hr_cae_event/models/event.py (hunk 1 above)
Comment (robinkeunen) [reply hr_cae_event/models/event.py:31]: I confirmed that with Catherine who said "one is enough"

File: hr_cae_event/views/event.xml (hunk 2 above)
Comment (robinkeunen) [reply hr_cae_event/views/event.xml:14]: 👌

**Comment by robinkeunen** [comment]
@mclaeysb indeed, let's see what Julien and Catherine have to say about adding he organisers to the attendees :/

**Comment by mclaeysb** [comment]
Merging this, eventhough some parts (notably on the co-organiser field) have not passed functional tests yet. If necessary we can bring it back or make a new branch!

**Pull request merged**